### PR TITLE
feat: SparkML MultilayerPerceptronClassifier

### PR DESCRIPTION
**Pull request by memoryz**
Signed-off-by: Jason Wang <jasowang@microsoft.com>

### CODE REVIEW

**Comment by memoryz** [comment]
#500

**Comment by memoryz** [comment]
Unit test model looks like this:
image

**Comment by memoryz** [comment]
@xadupre ready for review and merge.

File: onnxmltools/convert/sparkml/operator_converters/mlp_classifier.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        bias_vector = weights[offset : offset + layers[i + 1]]
+        offset += layers[i + 1]
+
+        if i == 0:
+            input = operator.inputs[0].full_name
+
+        weight_variable = scope.get_unique_variable_name("w")
+        container.add_initializer(
+            weight_variable,
+            onnx_proto.TensorProto.FLOAT,
```
Comment (xadupre) [inline onnxmltools/convert/sparkml/operator_converters/mlp_classifier.py:34]: A comment for future changes, it is possible to support double coefficients. It reduces the discrepancies.

**Review by xadupre — APPROVED** [review]

**Pull request merged**